packages/pagerduty-adapter/src/definitions/deploy/deploy.ts

```diff
@@ -215,6 +218,90 @@ const createCustomizations = (): Record<string, InstanceDeployApiDefinitions> =>
         },
       },
     },
+    [SCHEDULE_LAYERS_TYPE_NAME]: {
+      changeGroupId: deployment.grouping.groupWithFirstParent,
+      requestsByAction: {
+        customizations: {
+          add: [
+            {
+              request: {
+                endpoint: {
+                  path: '/schedules/{parent_id}',
+                  method: 'put',
+                },
+                context: {
+                  custom:
+                    () =>
+                    ({ change }) => ({
+                      time_zone: getParent(getChangeData(change)).value.time_zone,
+                    }),
+                },
+                transformation: {
+                  adjust: ({ value, context }) => ({
+                    value: {
+                      schedule: { schedule_layers: [value], time_zone: _.get(context, 'additionalContext.time_zone') },
+                    },
+                  }),
+                },
+              },
+              condition: {
+                custom:
+                  () =>
+                  ({ changeGroup, change }) =>
+                    !changeGroup.changes.some(
+                      scheduleChange =>
```

Comment:
you assume this is scheduleChange without checking it (although currently it is probably correct)
nit: 
```suggestion
                      change =>
```

Reply:
I know these are schedule changes as they are in the same change group

Reply:
currently, tomorrow there might be another change in this change group